cypress/integration/drView_spec.js

```diff
@@ -0,0 +1,71 @@
+describe('Doctors Display', () => {
+
+  beforeEach(() => {
+    cy.fixture('mockDr.json')
+      .then(mockDr => {
+        cy.intercept('GET', 'https://head-to-toe-be.herokuapp.com/api/v1/medical_professionals?type=doctor&state=South Dakota', {
+          statusCode: 201,
+          delay: 100,
+          body: mockDr
+        })
+      })
+    cy.visit('http://localhost:3000/')
+  });
+
+  it('Should be able to open to the main page', () => {
+  cy.url().should('eq', 'http://localhost:3000/')
+  });
+
+  it('Should acknowledge the presence of list of of doctors with a sub title', () => {
+    cy.get('#doctors-button').should('be.visible')
+      .get('#doctors-button').click()
+      .location('pathname').should('eq', '/doctors')
+
+      .get('.dr-sub-title').should('be.visible')
+      .get('.dr-sub-title').should('contain', 'Doctor')
+      .get('.dr-sub-title').should('have.css', 'font-family', 'opendyslexic')
+  });
+
+  it('Should display the doctors page when the doctor button is pressed', () => {
+    cy.get('#doctors-button').should('contain', 'Doctors')
+      .get('#doctors-button').click()
+      
+      .location('pathname').should('eq', '/doctors')
+      .get('h2').last().should('contain', 'Doctors here')
+  });
+
+  it('Should display the list of mhp providers for our users to see', () => {
+    cy.get('#doctors-button').should('be.visible')
+      .get('#doctors-button').click()
+      .location('pathname').should('eq', '/doctors')
+
+      .get('.provider-card').should('be.visible')
+      .get('.first-name').should('be.visible')
+      .get('.first-name').should('contain', 'sea lion')
+      .get('.last-name').should('contain', 'Drowzee')
+      .get('.address').should('contain', 'Raymon')
+    });
+
```

Comment:
This test seems to have a typo as it mentions mhp, when I think it was meant to be doctors. We also might want to add a test to this for number of children to show that multiple cards are being displayed

Reply:
Honestly, I love the test the children idea!